### DISCUSSION TITLE
Finished Currency Calculator

### DISCUSSION THREAD

**Discussion by StutzTAFE** [Felix-E-S/T24S2T1EXTG08#3] [General]
Hi Felix

It took awhile but I have finished the Currency Calculator so that it is in a working state.
The Aesthetics of the calculator can b discussed between us as of now it is very basic.
I'm not confident on how to push the repository to GitHub but will do some research first. 

Thanks,
Jethro

**Comment by StutzTAFE** [comment]
![image](https://github.com/user-attachments/assets/bb671e7a-f4d0-4c95-99b1-9a638c7bbd5b)
![image](https://github.com/user-attachments/assets/cf56c569-b51f-4ab9-ac7e-63b98edcab46)

Here are some images of it in use.

**Reply by Felix-E-S** [reply]
This is looking good! I will work on the mortgage calculator now. Let me know how you go with pushing your calculator.
